charts/workspacemanager/values.yaml

```diff
@@ -0,0 +1,21 @@
+# image is used for local Skaffold deploys
+# image: gcr.io/terra-kernel-k8s/terra-workspace-manager
+
+# appVerion is set by Helmfile
+# appVersion: 0.0.0
+
+imageConfig:
+  repository: gcr.io/terra-kernel-k8s/terra-workspace-manager
+  # Override the image tag to deploy by setting this variable.
+  # If no value is set, the chart's appVersion value will be used.
+  # tag: latest
+  imagePullPolicy: Always
+vault:
+  enabled: true
+  # pathPrefix is required if Vault secret sync is enabled
+  # pathPrefix: secret/dsde/terra/kernel/integration
+
+samAddress: https://sam.dsde-dev.broadinstitute.org/
```

Comment:
Nothing to do in this PR, but eventually we should change how services get addresses. That probably changes after we get Sam on K8s, though.